package.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
- [ ] Great job so far :+1: however there is an issue in your code, I can not compile your app or open it in the browser, :arrow_down: Kindly fix this 


![error1](https://user-images.githubusercontent.com/57562869/155035265-84feb52e-4a72-43f0-9eeb-9a07fd257ecb.png)
![error2](https://user-images.githubusercontent.com/57562869/155035267-91b9d290-1f89-4da2-aa2b-4897e73f4b1d.png)

- In the console

Reply:
Hello @moise10r
It's strange 🤔. My app runs normally here. 

![bookStore](https://user-images.githubusercontent.com/74314917/155035949-8abbfc7a-4cfc-43b5-9ff5-0ac9d2a9fc46.gif)

What do you think can cause this issue? 
I found something similar before and I solve it by deleting an external eslintrc.json

Reply:
Hello @moise10r 
I checked with my stand-up team. Thanks to @carloshs1994 in particular and it looks like the issue is caused because I'm ignoring the .babelrc file. I'm going to fix it now.
Thanks!

<img src="https://media2.giphy.com/media/f4V2mqvv0wT9m/giphy.gif"/>

Reply:
Great job :+1: @JohnFTitor